pkg/doc/sdjwt/common/common.go

```diff
@@ -18,7 +19,17 @@ import (
 )
 
 // DisclosureSeparator is disclosure separator.
-const DisclosureSeparator = "~"
+const (
+	DisclosureSeparator = "~"
+
+	SDAlgorithmKey = "_sd_alg"
+	SDKey          = "_sd"
+
+	disclosureParts = 3
+	saltIndex       = 0
+	nameIndex       = 1
```

Comment:
we always expect `Claim Name` binding?

Reply:
Yes, reference application is disclosing claims based on claim name.